upload-api/stores/provisions.js

```diff
@@ -4,16 +4,14 @@ import { CBOR, Failure } from '@ucanto/server'
 
 /**
  * Create a subscription ID for a given provision. Currently 
- * uses the CID of `customer` which ensures each customer
- * will get at most one subscription. This can be relaxed (ie,
- * by deriving subscription ID from customer AND consumer) in the future
- * or by other providers for flexibility.
+ * uses a CID generated from `customer` and `consumer` which ensures each customer
+ * will get at most one subscription per space.
```

Comment:
It'll also allow existing spaces that have already been provisioned to be provisioned again...can we fix/avoid?

Reply:
oh are you thinking of the case where someone else comes along and provisions a space I created? that would definitely create a thorny billing situation, though in the long run it would be nice to let more than one person sign up to pay for a space.

maybe we can just generate the billing ID from the space DID?

Reply:
oh HMMM good point, maybe this should just be generated from "consumer" so that a given space can only ever be provisioned once... will make that tweak and add a test!

Reply:
oh it turns out we already check for this in provisioning! I would like to leave the door open for a "multi-payer" space in the future so I think we should leave as currently implemented!